docs/source/download.rst

```diff
@@ -1,64 +1,86 @@
-=============
-Obtain Devito
-=============
+===========================
+Download and Install Devito
+===========================
+
+There are three main approaches to installing Devito.
+For those looking for a quick “out-of-the-box” way to try Devito, we recommend
+the route of `Docker installation`_. For those looking to use Devito as part of a project
+alongside other packages or/and develop for Devito, we suggest the `pip installation`_
+or the `conda installation`_.
 
-There are three main approaches to installing Devito. For those looking for the
-least-friction way to try Devito, we recommend the Docker route. For those
-looking to use Devito alongside other packages as part of a project, we support
-pip installation. If you're looking to develop for Devito, you might benefit from
-using the included conda environment that includes all the bells and whistles we
-recommend when developing for Devito.
 
 Docker installation
 -------------------
+
 You can install and run Devito via Docker_:
 
 .. _Docker: https://www.docker.com/  
 
+There are several "tags" for the Devito Docker images. We provide `Docker for CPUs`_
+and `Docker for GPUs`_ depending on the platform you want to deploy Devito. These are:
+
+Docker for CPUs
+```````````````
+Available images:
+
+- cpu-dev: for the latest development version (current GitHub master)
+- cpu-latest: for the latest GitHub release
+- cpu-vX.X.X: for a specific release (Release tags: https://github.com/devitocodes/devito/tags)
+
 .. code-block:: shell
 
-   # get the code
-   git clone https://github.com/devitocodes/devito.git
-   cd devito
+   # 1. Pull Devito image
+   docker pull devitocodes/devito:cpu-latest
+
+   # 2. (Optional but recommended) Run the tests
+   docker run --rm --name testrun 'devitocodes/devito:cpu-latest' pytest tests/test_operator.py
```

Comment:
The purpose of running the tests here is to "test installation". I agree that it is excessive to run all tests for this purpose (there is an open issue somewhere about this, I'll link it later if I can find it). With this PR, you are choosing test_operator.py to do that job of testing installation. I assume you have given this some thought. 
Edit: relevant issue: https://github.com/devitocodes/devito/issues/185

Reply:
I think that since this an Optional thing to do we may be fine...
Since no issue has been reported so far about this I feel safe...
Thoughts?